ingenialink/virtual/virtual_drive.py

```diff
@@ -22,91 +24,119 @@ class VirtualMonDistBase:
     """Base class to implement VirtualMonitoring and VirtualDisturbance.
 
     Args:
-        drive (VirtualDrive): Virtual drive instance.
+        drive: Virtual drive instance.
 
     """
 
     FREQUENCY = 20000
-    FREQ_DIVIDER_REG = None
-    BUFFER_SIZE_REG = None
-    NUMBER_MAP_REGS = None
-    MAP_REG_CFG = None
-    BYTES_PER_BLOCK_REG = None
-    AVAILABLE_BYTES_REG = None
-    DATA_REG = None
-
-    def __init__(self, drive):
+    FREQ_DIVIDER_REG = ""
+    BUFFER_SIZE_REG = ""
+    NUMBER_MAP_REGS = ""
+    MAP_REG_CFG = ""
+    BYTES_PER_BLOCK_REG = ""
+    AVAILABLE_BYTES_REG = ""
+    DATA_REG = ""
```

Comment:
Maybe you can simply write `DATA_REG: str`. Same with the others

Reply:
Done!